auto_hpl/base_test_results/test1/verify

```diff
@@ -0,0 +1,39 @@
+#
+# Main reg exp summary
+#
+#   Field 2: Test namere digits,
```

Comment:
typo, and fields out of order.

Also want text description of what is expected.  Applies to all.